core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -105,6 +105,7 @@
 #define CONT_HINT		(1ull << 0)
 
 #define UPPER_ATTRS(x)		(((x) & 0x7) << 52)
+#define GP                      BIT64(50)
```

Comment:
Since this file is for both Aarch32 LPAE and AArch64, how about adding a comment:
```
#define GP                      BIT64(50)  /* Guarded Page, Aarch64 FEAT_BTI */
```